receiver/prometheusreceiver/config.go

```diff
@@ -59,6 +59,11 @@ type Config struct {
 	UseStartTimeMetric   bool   `mapstructure:"use_start_time_metric"`
 	StartTimeMetricRegex string `mapstructure:"start_time_metric_regex"`
 
+	// UseCreatedMetric enables retrieving the start time for Summary, Histogram
+	// and Sum metrics from _created metric. It is ignored if UseStartTimeMetric
+	// is used.
+	UseCreatedMetric bool `mapstructure:"use_created_metric"`
```

Comment:
Should we make this a feature-gate instead of a flag?  Flags stay around forever vs feature-gates eventually go away (alpha = default off; beta = default on; ga = always on).

Reply:
Pushed the change. That is definitely better. I was not aware of this functionality.